walkabout-style.yaml

```diff
@@ -509,6 +509,7 @@ textures:
             biergarten: [780, 210, 38, 38]
             brewery: [780, 210, 38, 38]
             bike_bicycle: [962, 168, 38, 38]
+            bike_bicycle_rental: [964, 126, 38, 38]
```

Comment:
This change needs to be made also in the Illustrator file (so future exports don't revert).